app/javascript/components/DownloadButton.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faDownload } from '@fortawesome/free-solid-svg-icons';
+import Modal from 'react-bootstrap/lib/Modal';
+
+// TODO: Remove this once API endpoint is integrated
+import {authCodeResponseMock} from './FacetsMockData';
+
+function fetchDownloadConfig() {
+  const searchQuery = '&file_types=metadata,expression&accessions=SCP1,SCP2';
+  const authCode = authCodeResponseMock.auth_code; // Auth code is a one-time authorization token (OTAC)
+
+  const timeInterval = authCodeResponseMock.time_interval;
+
+  // Gets a curl configuration ("cfg.txt") containing signed
+  // URLs and output names for all files in the download object.
+  const url = (
+    window.origin +
+    '/api/v1/bulk_download?auth_code=' +
+    authCode +
+    searchQuery
+  );
+  const flag = (window.location.host === 'localhost') ? 'k' : ''; // "-k" === "--insecure"
```

Comment:
rename variable to curlSecureFlag?  Also think about checking the node environment (either at compile or runtime) instead of the hostname

Reply:
Done in 619e805.